src/services/SettingsService.ts

```diff
@@ -1,11 +1,19 @@
-import { BackendSiteSettings } from "types/settings"
+import {
+  BackendPasswordSettings,
+  BackendSiteSettings,
+  SitePasswordSettings,
+} from "types/settings"
 
 import { apiService } from "./ApiService"
 
 const getSettingsEndpoint = (siteName: string): string => {
   return `/sites/${siteName}/settings`
 }
 
+const getSettingsPasswordEndpoint = (siteName: string): string => {
+  return `/sites/${siteName}/settings/repo-password`
```

Comment:
hey alex i thought we were keeping the route camelCase? just wanted to check if we have changed both the route on FE & BE?

Reply:
I realised we already use some kebab-case endpoints (`/launch-site`), so it's already inconsistent! I swapped it on both FE and BE

Reply:
got it thanks!